packages/communication/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@celo/communication",
+  "version": "0.0.1-dev",
+  "description": "Celo's wrapper to interact with Celo network",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "author": "Celo",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/celo-org/celo-monorepo/tree/master/packages/communication",
+  "repository": "https://github.com/celo-org/celo-monorepo/tree/master/packages/communication",
+  "keywords": [
+    "celo",
+    "blockchain",
+    "defi"
+  ],
+  "scripts": {
+    "build": "tsc -b .",
+    "clean": "tsc -b . --clean",
+    "test": "jest --runInBand",
+    "lint": "tslint -c tslint.json --project ."
+  },
+  "dependencies": {
+    "@celo/utils": "0.1.20-dev",
+    "debug": "^4.1.1"
+  },
+  "devDependencies": {
+    "web3": "1.2.11",
+    "web3-core": "1.2.11",
+    "web3-eth": "1.2.11",
+    "web3-eth-contract": "1.2.11",
+    "web3-utils": "1.2.11",
```

Comment:
Why all these web3 items when `web3` includes all of them?

Reply:
My only concern with this, is that if I remove those versions, and for example, we are using another version of the submodules in one of our packages (and it will happen when I fix again the versions for the notification-service, the ODIS combiner and the faucet), yarn could use those libs instead of the ones of the Web3 1.2.11.
It happened to me before and I tried to avoid that.

Worst case scenario, it's redundant, because it will download only the web3 package, but this will surely avoid us cross-packages versions with already dowloaded libs